src/tests/ftest/util/command_utils.py

```diff
@@ -210,7 +211,14 @@ def stop(self):
                     self._command, str(state))
                 self._process.send_signal(signal_to_send)
                 if signal_list:
-                    time.sleep(5)
+                    start = time.time()
+                    try:
+                        self._process.wait(timeout=5)
```

Comment:
Here `self._process` is an `avocado.utils.process: class SubProcess(object)` object and  its `wait()` method can end up sending another `signal.SIGTERM` and even a `signal.SIGKILL` if the timeout is exceeded - something this loop is trying to control.

The general idea here is the `avocado.utils.process.SubProcess` provides useful methods - like `wait(timeout=None, sig=signal.SIGTERM)`, but they do not take into account remotely running processes, which is why this loop contains a call to `state = self.get_subprocess_state(msg)`.  This allows the command being executed, e.g. `daos_server`, via something like `orterun` to determine the state of the remotely running command.  By using `avocado.utils.process.SubProcess.wait()` we may send the `signal.SIGKILL` to the orterun process before being able to verify that the remotely running processes have been terminated.

In this case I think we want to call:
```suggestion
                        self._process._popen.wait(timeout=5)
```